src/charts/SimplePowerBIReportEmbed/SimplePowerBIReportEmbed.js

```diff
@@ -153,6 +160,32 @@ export const SimplePowerBIReportEmbed = ({
     }, refreshTimeout);
   };
 
+  const iframe = (
+    <PowerBIEmbed
+      cssClassName={classes.report}
+      embedConfig={embedConfig}
+      getEmbeddedComponent={(embedObject) => {
+        setReport(embedObject);
+      }}
+    />
+  );
+
+  let iframeContainer;
+  if (iframeRatio !== undefined && iframeRatio > 0) {
```

Comment:
Would it not be better to use `iframeRatio != null` in order to except _undefined_ and _null_ insteed _undefined_ only ?